tests/integration/suite/conftest.py

```diff
@@ -198,8 +198,8 @@ def user_factory(admin_mc, remove_resource):
     """
     def _create_user(globalRoleId='user'):
         admin = admin_mc.client
-        username = random_str()
-        password = random_str()
+        username = random_str() + "username"
+        password = random_str() + "password"
```

Comment:
Whats the point of this ?

Reply:
This is to ensure that username and password really are distinct. Although the chances of the following being the same are extremely low, the time segment of a username/password is almost always going to be the same because of int rounding. I'm also almost certain that this did happen while I was testing, but I don't have the error message anymore.
https://github.com/rancher/rancher/blob/7dcdd26018d8815f4dff7db1d5a318141b3f77a4/tests/integration/suite/common.py#L7-L8